browser/cypress/e2e/home_page.cy.ts

```diff
@@ -0,0 +1,102 @@
+// Sample test for home page
+
+describe('Home Page', () => {
+  // make sure basics of the page succesfully load
+  it('Home page loads successfully', () => {
+    cy.visit('/')
+
+    // Make sure the page has the landing splash
+    cy.contains('Genome Aggregation Database')
+    cy.contains('gnomAD browser')
+
+    // Make sure page content headers are there
+    cy.contains('New to gnomAD?')
+    cy.contains('About')
+  })
+
+  // json for each header page for ease of iterating for tests
+  const headerPages = {
+    pages: [
+      {
+        pageTitle: 'About',
+        pageUrl: '/about',
+        pageHeader: 'About gnomAD',
+      },
+      {
+        pageTitle: 'Team',
+        pageUrl: '/team',
+        pageHeader: 'The gnomAD Team',
+      },
+      {
+        pageTitle: 'Downloads',
+        pageUrl: '/downloads',
+        pageHeader: 'Google Cloud Public Datasets',
+      },
+      {
+        pageTitle: 'Policies',
+        pageUrl: '/policies',
+        pageHeader: 'gnomAD Ethics Policy',
+      },
+      {
+        pageTitle: 'Publications',
+        pageUrl: '/publications',
+        pageHeader: 'publications by the gnomAD group',
+      },
+      {
+        pageTitle: 'Feedback',
+        pageUrl: '/feedback',
+        pageHeader: 'Tell us how you use gnomAD',
+      },
+      {
+        pageTitle: 'Help',
```

Comment:
Can confirm that the Cypress test that checks for a properly rendered `pageHeader` at `/help` is failing... which is the true state of the world. Very cool!

![image](https://user-images.githubusercontent.com/8867295/183997778-ba4785c3-d883-4be5-967b-35ea0375c071.png)